### PR TITLE
Updated set 6 notebooks to fix the typo issue

### LINKED ISSUE #6: Typo in Set 6: `Set 6` mentioned instead of  `Set 7` at the end of the exercise and solution notebook
**Describe Error**
There is a Typo at the end of both the exercise and solution notebook where `Set 7` is inadvertently mentioned as `Set 6`.

```
✅ This completes Set 6: Grouping Methods (Exercises 51-60) of DatatableTon: 💯 datatable exercises
Set 6 is coming soon!
```

**Exercise / Set**
Set 6

**Suggested Fix**
```
✅ This completes Set 6: Grouping Methods (Exercises 51-60) of DatatableTon: 💯 datatable exercises
Set 7 is coming soon!
```

**Additional Comments**
Any other information about the error.

### PR DESCRIPTION
**Pull request by shrinidhin**
Closes Issue Typo in Set 6

### CODE REVIEW

File: notebooks/06_grouping_methods_exercises.ipynb [hunk 1]
```diff
@@ -696,7 +696,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.10"
+   "version": "3.9.6"
```
Comment (vopani) [inline notebooks/06_grouping_methods_exercises.ipynb]: The version shouldn't change in both notebooks.

**Review by vopani — CHANGES_REQUESTED** [review]

**Comment by shrinidhin** [comment]
Updated with requested changes

File: notebooks/06_grouping_methods_exercises.ipynb (hunk 1 above)
Comment (shrinidhin) [reply notebooks/06_grouping_methods_exercises.ipynb]: Updated notebook with the requested changes.

**Review by vopani — APPROVED** [review]
Thanks a lot!

**Pull request merged**